src/cider/nrepl.clj

```diff
@@ -8,52 +8,57 @@
             [cider.nrepl.middleware.pprint :as pprint]
             [cider.nrepl.print-method]))
 
-(def DELAYS (atom nil))
+(def DELAYS
```

Comment:
Why not `deferred-middleware`? Seems like the best possible name to me.

Reply:
Those are not middleware as such, but handlers which are deferred and that. It's also a map of `delay` objects, thus `DELAYS`.  `delayed-handlers` is probably a better name.